sdk/identity/azure-identity/tests/managed-identity-live/appservice.md

```diff
@@ -139,8 +141,8 @@ az acr login -n $ACR_NAME
 ### Set a variable for the image name
 
 ```sh
-export IMAGE_NAME=$(az acr show -n $ACR_NAME --query loginServer -o tsv)/webapp-managed-id-test \
-    PYTHON_VERSION=2.7
+export IMAGE_NAME=$(az acr show -n $ACR_NAME --query loginServer -o tsv)/webapp-managed-id-test  \
```

Comment:
It would make sense to tag the images with the Python version. Really you could copy the image building instructions from the AKS walkthrough verbatim, because these tests use the same image.

Reply:
Yes. The full image name is $IMAGE_NAME:$PYTHON_VERSION